events/elections/2020/README.md

```diff
@@ -31,24 +31,39 @@ previous [governance meeting video] which led to this whole process.
 
 ## Eligibility
 
-Please refer to the [Steering Committee Election Charter] for:
+Please refer to the [Steering Committee Election Charter] for [Eligibility for candidacy]
 
-- [Eligibility for candidacy]
-- [Eligibility for voting] and [Eligible voters] list (not in charter)
+Eligibility for voting in 2020 is defined as:
+
+* People who had at least 50 contributions to the Kubernetes project over
+  the past year, according to a snapshot taken 2020-08-01 of the data driving the [devstats developer activity counts dashboard][devstats-dashboard], who are also [Org Members].
+  Contributions include GitHub events like creating issues, creating pr's,
+  reviewing PR's, commenting on issues, etc. For full details see
+  [the SQL query used by devstats for developer activity counts][devstats-sql].
+
+* People who have submitted the [voting exception form] and are accepted by
+  the election committee. We *explicitly* believe the above heuristic will be
+  inaccurate and not represent the entire community. Thus we provide the form
+  for those who have contributed to the project but may not meet the above
+  criteria.  Acceptance of a form submission will be defined by a simple
+  majority vote, and the criteria used during this process will be used to
+  help refine further elections.
+
+The requirement to be an [Org Member] is new this year, added by the Steering Committee to ensure that voters are following community issues.  If you otherwise qualify to vote but have not yet applied for Org Membership, then please [request an exception][voting exception form].
 
 Corporate affiliation is applied after the election. If an organization finds itself with too many representatives it is up to those individuals to come to a consensus on who should serve on the committee.
 
 ### Schedule
 
 | Date         | Event                    |
 | ------------ | ------------------------ |
-| August XX    | Announcement of Election |
-| August XX    | Steering Committee Meeting with Q+A with the candidates and community | 
-| September XX | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
-| ~1 week      | Election prep week (voters.md validation and CIVS setup and testing) 
-| September XX | Election Begins via email ballots |
-| October XX   | Election Closes by 0000 UTC (5pm PST) |
-| October XX   | Announcement of Results at Community Meeting |
+| August 12    | Announcement of Election and publication of Voters.md |
+| August TBD    | Steering Committee Meeting with Q+A with the candidates and community |
+| September 08 | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| ~1 week      | Election prep week (voters.md validation and CIVS setup and testing)
+| September 14 | Election Begins via email ballots |
+| October 06   | Election Closes by 0000 UTC (5pm PST) |
+| October 12   | Announcement of Results at Public Steering Committee meeting |
 
 ## Candidacy Process
```

Comment:
GitHub doesn't allow me to comment on the "Nomination" section, so adding a comment here.

From last year's retro (https://github.com/kubernetes/steering/issues/140), there was a suggestion to move the nomination to GitHub. Specifically:

> Perhaps consider moving nominations to github, therefore org members are clearly marked and it would let people do reaction emojis and all the personal non-binding +1’s to not spam the development mailing list.
We could also ask people to clarify binding vs non-binding

Maybe we could ask the nominees to create a GitHub issue, ask them to send a link to the issue to k-dev and have endorsers +1 on the issue?

Reply:
Also from the retro:

> Feedback from/to nominee: would be great to chime in once enough official +1s have been reached

Can we add this to the process? Once the required number of +1s have reached, an election officer adds a comment stating so on the GitHub issue.

Reply:
+1 to (all but...) election officer letting folks know when the required number is reached is a good change to implement this round

Reply:
We discussed this today (it's the first time the current EC has heard of the idea), and while a good idea, it's too late to consider implementing it at this stage.    We would need to postpone the whole election schedule for a week, and I don't think it's worth it.

Reply:
+1 to what Josh said. We're already doing our best to keep things on track as-is.

Reply:
if gh is not an option, can you document that you will comment on the email threads when someone has reached the desired criteria/sponsorship?

it would be good to document here too that this is the last year that we will do this process as next year we will move to GitHub. this was a change that the majority of steering wanted to see this round but we are ok to bump to next year as long as there is an artifact in place so that its not to be missed again.